web-app/src/components/Bibles/BibleTable.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableContainer from "@material-ui/core/TableContainer";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import PropTypes from "prop-types";
+import VolumeOffIcon from "@material-ui/icons/VolumeOff";
+import VolumeUpIcon from "@material-ui/icons/VolumeUp";
+
+import Paper from "@material-ui/core/Paper";
+import ModalBox from "./ModalBox";
+
+const useStyles = makeStyles({
+	table: {
+		minWidth: 650,
+	},
+});
+
+const BibleTable = ({ bibles }) => {
+	const classes = useStyles();
+	const [book, setBook] = useState("");
```

Comment:
book can be called in the common context and the use effect also, then using useContext it can be fetched here in BibleTable.js